tests/conftest.py

```diff
@@ -0,0 +1,8 @@
+import pygfx
+
+
+MAX_TEXTURE_SIZE = 1024
+
+
+def pytest_sessionstart(session):
+    pygfx.renderers.wgpu.set_wgpu_limits(**{"max-texture-dimension2d": MAX_TEXTURE_SIZE})
```

Comment:
@clewis7 This is how we set the limit before the test session starts. The limit must be set before any renderer object is created, so it that's why it's in `pytest_sessionstart()`